pkgs/development/python-modules/keras/default.nix

```diff
@@ -5,12 +5,13 @@
 }:
 
 buildPythonPackage rec {
-  pname = "Keras";
-  version = "2.4.3";
+  pname = "keras";
+  version = "2.6.0";
+  format = "wheel";
```

Comment:
Can't we build this from source?

Reply:
Not easily, keras is built with bazel so it's a huge PITA to build from source. I propose that we start with the wheel build, and transitioning to building from source can be done in a separate PR.